lib/new_relic/rack/browser_monitoring.rb

```diff
@@ -69,30 +69,33 @@ def should_instrument?(env, status, headers)
 
       private
 
-      def autoinstrument_source(response, headers, js_to_inject)
+      def autoinstrument_source(response, _headers, js_to_inject)
         source = gather_source(response)
         close_old_response(response)
-        return nil unless source
+        return unless source
 
+        modify_source(source, js_to_inject)
+      rescue => e
+        NewRelic::Agent.logger.debug("Skipping RUM instrumentation on exception: #{e.class} - #{e.message}")
+      end
+
+      def modify_source(source, js_to_inject)
         # Only scan the first 50k (roughly) then give up.
         beginning_of_source = source[0..SCAN_LIMIT]
         meta_tag_positions = find_meta_tag_positions(beginning_of_source)
         if body_start = find_body_start(beginning_of_source)
           if insertion_index = find_insertion_index(meta_tag_positions, beginning_of_source, body_start)
             source = source_injection(source, insertion_index, js_to_inject)
           else
-            NewRelic::Agent.logger.debug("Skipping RUM instrumentation. Could not properly determine location to inject script.")
+            NewRelic::Agent.logger.debug('Skipping RUM instrumentation. Could not properly determine location to ' \
+                                         'inject script.')
```

Comment:
Why add` \ ` in the debug message?

Reply:
Ah, so while I was fixing the `AbcSize` issue I decided to also address another RuboCop issue with this code block to do with line length. By using backslashes, I just broke the super long debug text up into 120 column chunks.

Reply:
Oh just a length thing! Cool thank you